.vsts-dotnet-ci.yml

```diff
@@ -21,7 +21,15 @@ phases:
     name: Windows_NT
```

Comment:
Can you update this to be `Windows_NT_x64`?

Reply:
Also, I would just drop `NT`. It's the only `Windows` we know.

---
In reply to: [226813235](https://github.com/dotnet/machinelearning/pull/1309#discussion_r226813235) [](ancestors = 226813235)